docs/fastdds/getting_started/simple_python_app/includes/dataType.rst

```diff
@@ -52,7 +52,6 @@ This must have generated the following files:
     * HelloWorldCdrAux.hpp: C++ header file for HelloWorldCdrAux.ipp.
     * CMakeLists.txt: CMake file to generate C++ source code and Python module from the `SWIG`_ interface files,
       compile and generate C++ libraries.
-    * HelloWorld.py: Python module to be imported by your Python example.
 
```

Comment:
Hi @mirusu400 thanks for your contribution.

Instead of simply deleting it from the documentation, maybe we can explain better the actual behavior:

```suggestion

After that, the python bindings can be generated by running the following command.

.. code-block:: bash

    cmake .
    make

This must have generated the python binding:

    * HelloWorld.py: Python module to be imported by your Python example.
```